src/Css.elm

```diff
@@ -7948,3 +7950,170 @@ fill :
     -> Style
 fill (Value val) =
     AppendProperty ("fill:" ++ val)
+
+
+
+-- STROKE --
+
+
+{-| Sets [`stroke`](https://css-tricks.com/almanac/properties/s/stroke/)
+**Note:** `stroke` also accepts the patterns of SVG shapes that are defined inside of a `defs` element.
```

Comment:
I'll add the documentation. In the CSS tricks link there is already an example of how to use it. Using `def` you can define pattern in your SVG element and refer to the ID set by the user. So this would result into `url(#pattern)`. Now that I think about it maybe a better solution is to create a new Value called `pattern` that will always output `url(#pattern)` so other types of URL's are not allowed.